sktime/data/datasets.py

```diff
@@ -45,3 +48,50 @@ def ellipsoids(laziness=0.97, seed=None):
         an object that contains methods to create discrete and continuous observations
     """
     return Ellipsoids(laziness=laziness, seed=seed)
+
+
+def pbf(n_burn_in=5000, n_jobs=None):
+    r""" Creates a position based fluids :cite:`macklin2013position` simulator.
+
+    The simulation box has dimensions :math:`[-40, 40]\times [-25, 25]` and the initial positions are positioned
+    around the top boundary of the box. The interaction distance is set to :math:`d = 1.5` and `n_burn_in` steps are
+    performed to equilibrate the system before returning the simulator.
```

Comment:
I would love more details about how the # of particles (which I think are # of initial positions?) is determined

Reply:
the `datasets.py` file provides a top-level interface to the simulator. Here, the simulation domain as well as number of particles (in terms of initial positions) are fixed. In case custom positioning is wanted, one should refer to the simulator. I have clarified this in the docstring now.

Reply:
yay